### ISSUE TITLE
Validation loss is NaN 

### ISSUE THREAD

**Issue by mekaneeky** [SunbirdAI/parallel-text-EDA#5]
On training with acholi data, transfer learning from [flax-community/gpt2-swahili](https://huggingface.co/flax-community/gpt2-swahili) the model trains and begins to display a training loss by the 1st or 2nd evaluation step, however validation loss is consistently nan.

Requesting review as I suspect a dataset object creation error. @ak3ra
